spec/features/user_spec.rb

```diff
@@ -1,31 +1,50 @@
-require 'capybara/rspec'
-require 'capybara'
-require 'selenium-webdriver'
-require_relative '../../spec/spec_helper'
-require_relative '../../pages'
-require_relative '../../locators'
+require_relative '../spec_helper'
 
 feature 'OpenSourceCMS test', type: :feature do
 
-  describe Pages::Users do
-    it "#add_new_user" do
-      session = Pages::Users.new
-      session.open_main_page
-      session.visit_users_page
-      session.add_new_user('John', 'test@example.com', 'John', 'Smith', 'google.com', 'Password<>12348989898989')
+  describe 'When on Users Page' do
+    before(:each) do
+      @users_page = Users.new
+      @users_page.open_main_page
+      @users_page.visit_users_page
+    end
 
+    it 'Verifies user can be created' do
+      @users_page.add_new_user('John', 'test@example.com', 'John', 'Smith', 'google.com', 'Password<>12348989898989')
       expect(page).to have_content("New user created.")
     end
-  end
 
-  describe Pages::Users do
-    it "#delete_user" do
-      session = Pages::Users.new
-      session.open_main_page
-      session.visit_users_page
-      session.delete_user('John')
+    it 'Verifies possibility of editing existing user' do
+      @users_page.choose_user('John')
+      @users_page.edit_user('Author', 'Mark', 'Mich')
+      expect(page).to have_content("User updated.")
+    end
 
+    it 'Verifies user can be deleted' do
+      @users_page.delete_user('John')
       expect(page).to have_content("User deleted.")
     end
   end
+
+  describe 'Negative user stories when on Users Page' do
+    before(:each) do
+      @users_page = Users.new
+      @users_page.open_main_page
+      @users_page.visit_users_page
+    end
+    it "Verifies user can't be created if username is already exists" do
+      @users_page.add_new_user('Michael', 'test@test.com', 'John', 'Smith', 'google.com', 'Password<>12348989898989')
+      expect(page).to have_content("This username is already registered. Please choose another one.")
+    end
+
+    it "Verifies user can't be created if e-mail is already exists" do
```

Comment:
It is nice chance to try and combine this and the above scenarios, since they are the same apart from test data.